spec/jobs/file_upload_job_spec.rb

```diff
@@ -9,6 +9,14 @@
       allow(FileWorker).to receive(:new).and_return(instance_double(FileWorker, send: true))
     end
 
+    describe "concurrency control" do
+      it "includes limits_concurrency declaration in job class" do
+        job_source = File.read(Rails.root.join("app/jobs/file_upload_job.rb"))
```

Comment:
The same note here about testing letters

Reply:
Removed that spec, it was testing nothing but the spec itself :sob: